.github/workflows/ost.yml

```diff
@@ -5,7 +5,19 @@ on:
     types: [created]
 
 jobs:
+  parse-params:
+    outputs:
+      ost_suite: ${{ steps.parse-params.outputs.ost_suite }}
+      ost_images_distro: ${{ steps.parse-params.outputs.ost_images_distro }}
+    steps:
+    - id: parse-params
+      run: |
+        echo "::set-output name=ost_suite::$(echo ${{ github.event.comment.body }} | egrep -o '\w+-suite-master' || echo basic-suite-master)"
+      run: |
+        echo "::set-output name=ost_images_distro::$(echo ${{ github.event.comment.body }} | egrep -o '(el8stream|el9stream|rhel8)' || echo el8stream)"
```

Comment:
github yaml syntax is a good programming environment. If you want to have some
logic in ci script, write proper python script to parse it.

Also this is needed for all projects, no? why vdsm should keep special code for ost?

Reply:
If the logic was kept in a python script we'd need to create it inline in the job definition.

We still don't have a solution for reusing this logic across oVirt projects. I'll be happy to move this code once someone figures out how to do this.

Reply:
> We still don't have a solution for reusing this logic across oVirt projects.

As announced by Michal, we have one now, so this PR is not needed anymore. And we can even simplify Vdsm ost.yml workflow, based on https://github.com/oVirt/ovirt-system-tests/blob/master/.github/example-trigger.yaml, right?